webapp/channels/src/components/textbox/textbox.tsx

```diff
@@ -274,11 +274,11 @@ export default class Textbox extends React.PureComponent<Props> {
             textboxClassName += ' textarea--has-labels';
         }
 
-        if (this.props.preview) {
-            return (
+        return (
+            <>
                 <div
                     ref={this.wrapper}
-                    className='textarea-wrapper'
+                    className={classNames('textarea-wrapper', {'display-none': !this.props.preview})}
```

Comment:
i am wary of this approach because even though we are hiding the dom element react still renders both component.  We might want to check out the disable animation approach @hmhealey mentioned

Reply:
The challenge is disabling the transition for the specific case when preview is disabled, as transition is needed for other cases. This would require observing the specific state change and disabling the transition in the textarea element which is nested deep inside.  Do you have any suggestions 🤔  ?

Reply:
I'm actually good with this solution. It's actually the alternate solution I was going to propose since that's how it used to work before #23973 where I spent some time fighting with making this work. I thought I'd actually reverted that part of the PR, but I guess I didn't